### DISCUSSION TITLE
How to Train the Flux-Fill-dev Model? Issues with Latent and Mask Concatenation

### DISCUSSION THREAD

**Discussion by yyyyyxie** [bghira/SimpleTuner#1203] [Q&A]
Hi,
I want to training the flux-fill-dev, but I’m unsure where to start.  Specifically, I always seem to get the latent representations using VAE, but I’m unsure how to concatenate the latent and the mask according to the channel concatenation method described in flux-fill-dev.

**Comment by bghira** [comment]
i dont support ip adapter or inpainting training

**Reply by bc129697** [reply]
Hi, Can I use image pairs(raw_image+mask) to train flux-fil-dev model?

**Comment by yyyyyxie** [comment]
Thank you for your help. How I can save the quantized weight files obtained using your framework, specifically in the safetensors format?

**Comment by bghira** [comment]
do you mean the int8 or fp8 versions created at startup? currently you cannot.

**Comment by cardosofelipe** [comment]
I am very interested in this topic @yyyyyxie did you managed to train loras with flux-fill?
